### PR TITLE
First half of product page

### PR DESCRIPTION
**Pull request by dalisc**
This pull requests contains code that was merged from Evelyn's branch. To minimise merge conflicts due to the little time we have left, only I will be doing the final merges for the product-page.

Changes
- Image gallery lightbox
- Update top portion (the logo and stars)
- Organised files: Split product.html css and assets into separate files

Remaining:
- About
- Menu Description
- Reviews

### CODE REVIEW

File: src/main/webapp/product.html [hunk 1]
```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Product Listing</title>
+    <link rel="stylesheet" href="assets/css/product-page.css" />
+    <link rel="stylesheet" href="assets/css/lightbox.css" />
+    <script src="assets/js/lightbox.js"></script>
```
Comment (g-lilian) [inline src/main/webapp/product.html:7]: The directory structure is bugged! I think you meant to put js in the same level as assets, not under css? When I ran the server, it showed Failed to load resource.
Screenshot_1

Comment (dalisc) [reply src/main/webapp/product.html:7]: oh yes thank you! no wonder why my js wasn't working! i thought i would debug it proper when cloud shell is up (it's been down). I'll make changes

**Comment by dalisc** [comment]
> Nice image gallery! Are we having a separate page with a long list of different shops? This page is the maximised view for a single shop right? I was wondering what page the 'Browse listings' tab in the nav bar is supposed to point to.

Yes! We have two separate pages. The one you're working on with Jireh and Jacob is the one with the different shops and the one I'm working on with Evelyn is the details of a single shop. I will incorporate the navbar into this page too so that we can go back to Browse listings

**Comment by g-lilian** [comment]
> Yes! We have two separate pages. The one you're working on with Jireh and Jacob is the one with the different shops and the one I'm working on with Evelyn is the details of a single shop. I will incorporate the navbar into this page too so that we can go back to Browse listings.
> 
> UPDATE: I see that your Browse Listings in the navbar redirects to listings.html. I don't think this is necessary. Perhaps for the navbar we should just have Home (redirect to index.html) and About (redirect to about.html)

Ohh I see, will make the changes to the nav bar then. I suppose it makes more sense to have some listings in the front page and a page displaying relevant listings after a user search, rather than a separate long page of listings.

**Pull request merged**